InvenTree/plugin/base/integration/AppMixin.py

```diff
@@ -36,7 +36,6 @@ def _activate_mixin(cls, registry, plugins, force_reload=False, full_reload: boo
 
         if settings.PLUGIN_TESTING or InvenTreeSetting.get_setting('ENABLE_PLUGINS_APP'):
             logger.info('Registering IntegrationPlugin apps')
-            apps_changed = False
```

Comment:
This adds to the testing time significantly - when we added the check for testing it remove around 5 mins of runtime. With the reduced overall time the impact might be lower but needlessly reloading every test init seems like a waist.
If you want to remove the change check better replace it with a flag for testing